src/js/components/Marathon.jsx

```diff
@@ -49,8 +50,10 @@ var Marathon = React.createClass({
   componentWillMount: function () {
     // TODO: That should be handled directly on the NavTabs
```

Comment:
Can you create a ticket for this leftover todo, or just remove the line if doesn't apply anymore?